<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGinTonic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Gin &amp; Tonic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSomaek">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Somaek</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOrangeTree">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Orange Tree</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGinCoke">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Gin Coke</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDiesel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Diesel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pump_1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>1번 펌프</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pump_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>240</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>2번 펌프</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pump_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>3번 펌프</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pump_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>310</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>4번 펌프</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pump_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>380</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>5번 펌프</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pump_6">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>380</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>6번 펌프</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnpower">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Power ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enstop">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>E -Stop</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
